{"name": "Ошибка 500 Internal Server Error при попытке сделать заказ авторизованным пользователем с невалидным хешем ингредиента", "status": "passed", "description": "При попытке сделать заказ авторизованным пользователем ингредиенты с невалидным хешем, возвращается ошибка 500 Internal Server Error", "steps": [{"name": "Создание заказа", "status": "passed", "parameters": [{"name": "headers", "value": "{'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3OTE1MjI2OWVkMjgwMDAxYjU1ODQzNCIsImlhdCI6MTczNzU3Njk5OCwiZXhwIjoxNzM3NTc4MTk4fQ.FoMTDsKgt1x__3J-_oOMdPdDoa7fs47Gm7Nv02UbS8I'}"}, {"name": "ingredients", "value": "{'ingredients': ['777kit555', '34ter434']}"}], "start": 1737576999160, "stop": 1737576999228}], "start": 1737576999159, "stop": 1737576999229, "uuid": "46443d32-c6aa-4f60-8b5e-0847d692573c", "historyId": "9eeb56360e08e7affc6efa227e1428c6", "testCaseId": "9eeb56360e08e7affc6efa227e1428c6", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_auth_user_wrong_hash_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "01ms32"}, {"name": "thread", "value": "20896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}